x-pack/plugins/security_solution/public/detections/containers/detection_engine/alerts/use_host_isolation_status.tsx

```diff
@@ -65,7 +64,8 @@ export const useHostIsolationStatus = ({
           setPendingUnisolation(data[0].pending_actions?.unisolate ?? 0);
         }
       } catch (error) {
-        addError(error.message, { title: ISOLATION_PENDING_FAILURE });
+        // silently catch non-user initiated error
+        return;
```

Comment:
I wonder if we could be a bit more specific on the errors that remain silent in the future.

Reply:
i think if there are certain errors we want to catch / call out we can do so, i'm just not sure which ones we would want to call out i guess